### PR TITLE
Update excludes list for Windows

### PR DESCRIPTION
**Pull request by jongiddy**
## High-level description

Update excludes list for Windows, so that it can build. Also, adds some additional logging to `pkgpanda` which makes packages causing failures more obvious.

This does not completely fix the build, just moves it to the next problem, which is a Mesos build error.

See the current `master` state at https://teamcity.mesosphere.io/viewLog.html?buildId=2147666&tab=buildResultsDiv&buildTypeId=DcOs_Windows_Build_BuildDcOs which takes about 15 minutes to fail at the start of the build phase.

See this branch's state at https://teamcity.mesosphere.io/viewLog.html?buildId=2149125&tab=buildResultsDiv&buildTypeId=DcOs_Windows_Build_BuildDcOs which takes about 45 minutes to fail with a Mesos build error, having completed the initial build phase. The Mesos build error is reported as https://jira.mesosphere.com/browse/DCOS_OSS-5431

(latest results of these builds can be seen at https://teamcity.mesosphere.io/viewType.html?buildTypeId=DcOs_Windows_Build_BuildDcOs)

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5429](https://jira.mesosphere.com/browse/https://jira.mesosphere.com/browse/DCOS_OSS-5429) Fix Windows build packages
  - [DCOS-45547](https://jira.mesosphere.com/browse/DCOS-45547) teamcity/dcos/build/dcos/windows: 'package python is supposed to have variant windows'

## Related tickets (optional)

  - [DCOS-ID](https://jira.mesosphere.com/browse/DCOS-<number>) JIRA title / short description.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [ ] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
Build fails earlier if reverted.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by jongiddy** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jongiddy** [comment]
@mesosphere-mergebot request-review @jgehrcke @mhrabovcin

File: pkgpanda/build/__init__.py [hunk 1]
```diff
@@ -220,6 +220,8 @@ def package_tuples_with_dependencies(package_names, treeinfo, package_store):
     @staticmethod
     def validate_package_tuples(package_tuples, treeinfo, package_store):
         # Validate that all packages have the variant specified in treeinfo.
+        print('package_tuples = %r' % package_tuples)
```
Comment (mhrabovcin) [inline pkgpanda/build/__init__.py]: Are these lines debug output that shouldn't be printed in final build?

Comment (jongiddy) [reply pkgpanda/build/__init__.py]: These lines provide the information that lets us better identify the problem in the future. `print` is currently `pkgpanda`'s log method, at least on the build side.

**Comment by jongiddy** [comment]
@mesosphere-mergebot changelog-not-required fixing failed build

**Pull request merged**